__tests__/integration/profilesLoad.test.js

```diff
@@ -5,25 +5,112 @@ describe('Profiles load', () => {
     await page.goto('http://127.0.0.1:8000/__tests__/integration/index.html');
     await expect(page).toClick('a[href="#create"]', { text: 'Editor' })
     await page.waitFor(1000)
-  });
+  })
 
-  it('loads profile for Monograph Instance', async () => {
+  describe('loads profile for Monograph Instance', () => {
+    beforeAll(async () => {
+      const monograph_sel = 'ul.nav-sidebar > li:first-child > a:first-child'
+      await expect(page).toClick(monograph_sel, { text: 'Monograph' })
+      const instance_sel = 'a#sp-0_0'
+      await expect(page).toClick(instance_sel, { text: 'Instance' })
+      await page.waitFor(1000)
+    })
 
-    const monograph_sel = 'ul.nav-sidebar > li:first-child > a:first-child'
-    await expect(page).toClick(monograph_sel, { text: 'Monograph' })
+    it('displays profile title', async () => {
+      await expect_regex_in_selector_textContent('div#bfeditor-formdiv > form > div > h4', /^BIBFRAME Instance/)
+    })
+    it('displays appropriate input fields and labels', async () => {
+      // TODO: full input field functionality #54 - separate test file, perhaps
+      const edition_label_sel = 'label[title="http://access.rdatoolkit.org/2.5.html"]'
+      await expect_value_in_selector_textContent(edition_label_sel + ' > a', 'Edition Statement (RDA 2.5)')
```

Comment:
Instead of using the "+" to concatenate to make this selector, you could use Javascript template literal with back-ticks, `${edition_label_sel} > a`